### PR TITLE
docs(upgrade) change prometheus changelog and update prometheus dashboard

### PR DESCRIPTION
**Pull request by attenuation**
### Summary

1. update prometheus changelog to explain the new flag
2. update prometheus grafana json panel

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -228,7 +228,7 @@
   [#8712](https://github.com/Kong/kong/pull/8712)
 - **Prometheus**: The plugin doesn't export status codes, latencies, bandwidth and upstream
   healthcheck metrics by default. They can still be turned on manually by setting `status_code_metrics`,
-  `lantency_metrics`, `bandwidth_metrics` and `upstream_health_metrics` respectively.
+  `lantency_metrics`, `bandwidth_metrics` and `upstream_health_metrics` respectively. Because of these flag function will cause the perfmance impact, we recommend using the new `status` plugin to replace it. And now `prometheus` plugin new grafana [dashboard](https://grafana.com/grafana/dashboards/7424-kong-official/) updated
```
Comment (guanlan) [inline CHANGELOG.md:231]: perfmance -> performance

Comment (guanlan) [reply CHANGELOG.md:231]: status -> statsd

Comment (guanlan) [reply CHANGELOG.md:231]: "Enabling those metrics will impact the performance if you have a large volume of Kong entities, we recommend using the statsd plugin with the push model if that is the case."

**Review by fffonion — APPROVED** [review]

**Comment by attenuation** [comment]
grafana dashboard screenshot is 
image
photo_2022-09-09 17 24 38

**Comment by chronolaw** [comment]
Do not forget to cherry-pick to branch `3.0.x`

**Pull request merged**